gitlab/resource_gitlab_user.go

```diff
@@ -105,6 +111,7 @@ func resourceGitlabUserSetToState(d *schema.ResourceData, user *gitlab.User) {
 	d.Set("is_admin", user.IsAdmin)
 	d.Set("is_external", user.External)
 	d.Set("note", user.Note)
+	d.Set("state", user.State)
 }
 
 func resourceGitlabUserCreate(ctx context.Context, d *schema.ResourceData, meta interface{}) diag.Diagnostics {
```

Comment:
Can you add this to the "create" function _after_ the call to `d.SetId` --

```go
if d.Get("state") == "blocked" {
	if err := client.Users.BlockUser(id); err != nil {
		return diag.FromErr(err)
	}
}
```

Reply:
Added